doc/cabal-package.rst

```diff
@@ -903,22 +912,21 @@ section on `build information`_).
 
 .. _sublibs:
 
-**Sublibraries**
+**Sublibraries - Examples**
 
-Cabal 2.0 and later support "sublibraries", which are extra named
-libraries (as opposed to the usual unnamed library section). For
-example, suppose that your test suite needs access to some internal
-modules in your library, which you do not otherwise want to export. You
-could put these modules in a sublibrary, which the main library
-and the test suite :pkg-field:`build-depends` upon. Then your Cabal file might
-look something like this:
+An example of the use of an internal library (that is, a 'private' sublibrary)
+is a test suite that needs access to some internal modules in your 'main'
+unnamed library, which you do not otherwise want to expose. You could put those
```

Comment:
```suggestion
is a test suite that needs access to some internal modules in your main
library, which you do not otherwise want to expose. You could put those
```
As above

Reply:
Reflected in my subsequent changes.